app/src/main/java/openfoodfacts/github/scrachx/openfood/analytics/AnalyticsTrackingEvent.kt

```diff
@@ -0,0 +1,22 @@
+package openfoodfacts.github.scrachx.openfood.analytics
+
+import android.app.Activity
+import java.util.concurrent.TimeUnit
+
+fun Activity.startTrackEvent(event : AnalyticsEvent): AnalyticsTrackingEvent {
+    return AnalyticsTrackingEvent(event)
+}
+
+data class AnalyticsTrackingEvent(
+    val startDate: Long,
```

Comment:
```suggestion
    val startDate: Long = System.currentTimeMillis(),
```
and remove the secondary constructor